src/command/info/info.c

```diff
@@ -1290,19 +1429,43 @@ infoRender(void)
 
                             // Output the status per repo
                             VariantList *repoSection = kvGetList(stanzaInfo, STANZA_KEY_REPO_VAR);
+                            String *formatSpacer = strNew("            ");
```

Comment:
This is set to something different later but I realized it might be clearer to put it all on one line, so did that in commit ed486fab81a4ac27406b0903a93fc9e62ba62b01

Reply:
OK, but you are still dynamically allocating strings here. Why not:
```
const char *formatSpacer = !multiRepo ? "            " : "               ";
```

Reply:
Although, I really prefer:
```
const char *formatSpacer = multiRepo ? "               " : "            ";
```

Reply:
Fixed, commit eae2307edf987b552a294e602273d9eceb694e14